torch/_higher_order_ops/flex_attention.py

```diff
@@ -499,15 +510,17 @@ def flex_attention_fake_impl(
     if query.is_nested:
         out = torch.empty_like(query, memory_format=torch.contiguous_format)
         logsumexp = query.sum(dim=-1)
-        return out, logsumexp
+        max_scores = query.sum(dim=-1)
+        return out, logsumexp, max_scores
 
     v_head_dim = value.size(-1)
     batch_size, num_heads, seq_len_q, _q_head_dim = query.shape
     logsumexp = query.new_empty(batch_size, num_heads, seq_len_q, dtype=torch.float32)
+    max_scores = query.new_empty(batch_size, num_heads, seq_len_q, dtype=torch.float32)
```

Comment:
I think we should resize to allocate 0 sized if we aren't going to use in order to save mem + runtime

Reply:
next pr